.gitignore

```diff
@@ -1,5 +1,6 @@
 build_logs/
 
+.gitmodules
```

Comment:
This should also not be ignored because we actually _have_ a .gitmodules file which needs to be there!
```suggestion
```